cookbooks/cdo-nodejs/attributes/default.rb

```diff
@@ -1,3 +1,3 @@
 default['cdo-nodejs']['version'] = '6.x'
 default['cdo-nodejs']['npm_version'] = '3'
-default['cdo-nodejs']['yarn_version'] = '0.16.1'
+default['cdo-nodejs']['yarn_version'] = '0.16.1-1'
```

Comment:
Just an naming issue with the available package versions on `apt`:

![image](https://cloud.githubusercontent.com/assets/1615761/21120043/3761c16e-c07a-11e6-95e0-465e82d1dd81.png)

![image](https://cloud.githubusercontent.com/assets/1615761/21120049/45ee73b2-c07a-11e6-93ad-f0af0ae037bd.png)